src/Http/Import/ImportTermRequestBodyParser.php

```diff
@@ -58,7 +59,11 @@ function ($term) use ($ignoreDomains) {
                 function ($term, int $index) {
                     if ($term instanceof stdClass && isset($term->id) && is_string($term->id)) {
                         $id = $term->id;
-                        $category = $this->categoryResolver->byId(new CategoryID($id));
+                        try {
+                            $category = $this->categoryResolver->byId(new CategoryID($id));
+                        } catch (EmptyCategoryId $exception) {
+                            $category = null;
```

Comment:
Should we catch this? We once discussed that we would not silently accept wrong values or modify them to correct values. We should always inform the API integrator with a problem report.

Reply:
They will get a 4xx Error Response, instead of a 5xx Server Error.
`The term  does not exist or is not supported` Same as one uses  a termid that does not exist(e.g., a term with id 123)

Reply:
As discussed we now report a more specific error about an empty category ID